operator/bundle_helpers/requirements-gha.txt

```diff
@@ -1,5 +1,5 @@
 # TODO(ROX-26860): remove this file and use just requirements.txt once the GHA operator build runs with Python 3.9.
 # PyYAML > 6.0 requires Python > 3.6.
-PyYAML==6.0
+PyYAML==6.0.1
```

Comment:
This is fine for me if all builds and CI passes. 
There was a reason to pin specifically `6.0`, but I don't remember it.
`>=6.0.2` will not work with the test image.